search-app/src/cmr/search/api/routes.clj

```diff
@@ -365,6 +376,11 @@
                {params :params headers :headers ctx :request-context}
                (get-deleted-collections ctx path-w-extension params headers)))
 
+        (context ["/:path-w-extension" :path-w-extension #"(?:collections-created-after)(?:\..+)?"] [path-w-extension]
```

Comment:
I did that because the existing api would not work. We don't have the capability to search for deleted collections through regular search. This is searching for visible collections which the regular search does support.